### PR TITLE
[IMP] website: introduce `s_references_social` snippet

### PR DESCRIPTION
**Pull request by mano-odoo**
This PR adds the new s_references_social snippet.

task-4146797
Part-of: task-4077427

Design-theme: https://github.com/odoo/design-themes/pull/890

| New s_references_Social |
|--------|
| image |
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_references_social.xml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_references_social" name="References Social">
+    <section class="s_references_social o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row">
+                <div>
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_references_social.xml:8]:
```suggestion
                <div class="col-lg-12">
```
Looks like it takes full width by default, but I think it's cleaner to declare it 👍

File: addons/website/views/snippets/s_references_social.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    <p class="lead" style="text-align: center;">We are in good company.</p>
+                </div>
+                <div class="col-lg-3 pt16 pb16">
+                    <p style="text-align: center;">
+                        <img class="img img-fluid" src="/web/image/website.s_reference_demo_image_1" alt=""/>
+                    </p>
+                    <h3 class="h5-fs" style="text-align: center;">Amsterdam</h3>
+                    <p style="text-align: center;">Fruitful collaboration since 2014</p>
+                    <div class="s_social_media o_not_editable text-center no_icon_color" data-snippet="s_social_media" data-name="Social Media">
+                        <h4 class="s_social_media_title d-none o_default_snippet_text">Social Media</h4>
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_references_social.xml:19]: Not sure we need this title, see: https://github.com/odoo/odoo/pull/179007/files#r1747123097

**Review by anso-odoo — CHANGES_REQUESTED** [review]
Good job, small remarks 👍

File: addons/website/views/snippets/s_references_social.xml (hunk 2 above)
Comment (mano-odoo) [reply addons/website/views/snippets/s_references_social.xml:19]: I leave it unresolved like https://github.com/odoo/odoo/pull/179007#discussion_r1747123097 for tech review 😄

File: addons/website/views/snippets/s_references_social.xml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            <div class="row">
+                <div class="col-lg-12">
+                    <h2 style="text-align: center;">Our valued partners</h2>
+                    <p class="lead" style="text-align: center;">We are in good company.</p>
+                </div>
+                <div class="col-lg-3 pt16 pb16">
+                    <p style="text-align: center;">
+                        <img class="img img-fluid" src="/web/image/website.s_reference_demo_image_1" alt=""/>
+                    </p>
+                    <h3 class="h5-fs" style="text-align: center;">Amsterdam</h3>
```
Comment (anso-odoo) [inline addons/website/views/snippets/s_references_social.xml:16]:
```suggestion
                    <img class="img img-fluid" src="/web/image/website.s_reference_demo_image_1" alt=""/>
                    <h3 class="h5-fs" style="text-align: center;"><br/>Amsterdam</h3>
```

I actually think it's better to do it like this. When the image is wrapped in a `<p>` and I try to delete the empty line (hitting backspace), the heading becomes a paragraph

**Review by anso-odoo — CHANGES_REQUESTED** [review]
Good job, just one comment

**Review by anso-odoo — APPROVED** [review]

**Review by bvr-odoo — APPROVED** [review]
LGMT :+1: Thanks

**Comment by SergeBayet** [comment]
Commit(s) moved into
- https://github.com/odoo/odoo/pull/180736

**Pull request closed without merging**